src/server/pachyderm_test.go

```diff
@@ -1337,6 +1337,56 @@ func TestRunPipeline(t *testing.T) {
 	//})
 }
 
+func TestPipelineJobHasAuthToken(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+	t.Parallel()
+	c, _ := minikubetestenv.AcquireCluster(t)
+	tu.ActivateAuthClient(t, c)
+	rc := tu.AuthenticateClient(t, c, auth.RootUser)
+
+	dataRepo := tu.UniqueString("TestPipelineJobAuthToken_data")
+	require.NoError(t, rc.CreateRepo(pfs.DefaultProjectName, dataRepo))
+
+	commit, err := rc.StartCommit(pfs.DefaultProjectName, dataRepo, "master")
+	require.NoError(t, err)
+	require.NoError(t, rc.PutFile(commit, "file", strings.NewReader("foo\n"), client.WithAppendPutFile()))
+	require.NoError(t, rc.FinishCommit(pfs.DefaultProjectName, dataRepo, commit.Branch.Name, commit.Id))
+
+	pipeline := tu.UniqueString("pipeline")
+	require.NoError(t, rc.CreatePipeline(pfs.DefaultProjectName,
+		pipeline,
+		"",
+		[]string{"bash"},
+		[]string{
+			"echo PACH_TOKEN=$PACH_TOKEN",
+			"echo $PACH_TOKEN | pachctl auth use-auth-token",
+			"pachctl config update context --pachd-address 'grpc://pachd.test-cluster-1.svc.cluster.local:30660'",
+			"pachctl auth whoami",
```

Comment:
maybe pipe this result into `/pfs/out/result.txt` and assert on JOB_SUCCESS that the contents of the file are the `pipeline:default/{pipeline}`?